src/TodoItems.js

```diff
@@ -1,30 +1,29 @@
 import React from 'react';
 import {
-    View,
-    Text,
-    FlatList,
-  } from 'react-native';
+  View,
+  Text,
+  FlatList,
+} from 'react-native';
 import { styles } from './styles.js';
 
 const items = [
-    {key: 'Cleaning'},
-    {key: 'Studying'},
-    {key: 'Read 40 pg of the Book'},
-    {key: 'Go for a walk with doggo'},
-    {key: 'Call mum'},
-    {key: 'Make dinner'},
-    {key: 'Exercise'},
-  ];
+  { key: 'Cleaning' },
+  { key: 'Studying' },
+  { key: 'Read 40 pg of the Book' },
+  { key: 'Go for a walk with doggo' },
+  { key: 'Call mum' },
+  { key: 'Make dinner' },
+  { key: 'Exercise' },
+];
 
-const renderToDoItem = ({item}) => (<Text style={styles.highlight}>{item.key}</Text>)
+const renderToDoItem = ({ item }) => (<Text style={styles.highlight}>{item.key}</Text>);
 
-export const TodoItems = () => {
-    return (
-      <View style={styles.sectionContainer}>
-        <FlatList
-          data={items}
-          renderItem={renderToDoItem}
-        />
-      </View>
-    );
-}
\ No newline at end of file
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
To avoid esliint disable in each component it is a good practice to change rules in global eslint config file like it is offered here https://stackoverflow.com/a/59166065